shell/platform/linux/fl_view.cc

```diff
@@ -51,6 +51,8 @@ struct _FlView {
 
   GList* children_list;
   GList* pending_children_list;
+
+  gboolean pointer_inside;
```

Comment:
Please add a comment above this variable.

Reply:
Done.